functions/contrib/ts/helm-inflator/src/gen/io.k8s.api.admissionregistration.v1beta1.ts

```diff
@@ -1,5 +1,5 @@
-import { KubernetesObject } from 'kpt-functions';
-import * as apisMetaV1 from './io.k8s.apimachinery.pkg.apis.meta.v1';
+import { KubernetesObject } from "kpt-functions";
```

Comment:
I suppose these quotes should have already checked by prettier. Why are they changed?

Reply:
I didn't copy over `functions/contrib/.prettierrc.json`. It uses single quote again.